pkg/rtc/uptrackmanager_test.go

```diff
@@ -257,7 +257,7 @@ func TestSubscriptionPermission(t *testing.T) {
 				},
 			},
 		}
-		um.UpdateSubscriptionPermission(subscriptionPermission, nil, nil, nil)
+		um.UpdateSubscriptionPermission(subscriptionPermission, utils.TimedVersion{}, nil, nil)
```

Comment:
guess I was lazy to add tests for skipping older version when I initially added the subscription permission version 😞 . Mind adding some tests for the timed version check paths.

Reply:
we defer to syncstore for version checking. emitted values are ordered and all of the updates are processed in a single goroutine so we don't need to check them in implementation code anymore.

Reply:
We should probably add tests or remove any version check completely from the code. 'SyncStore' is an implementation, but this code is released independent of 'SyncStore'. I think it is fine to say this module does not do any version check (looked like it still checks although I might have misread the diff) and it is an external responsibility to provide the right version. But if this module is doing version checks, would be good to add some tests. Not necessary to do now. Can be added later.